oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 server-tools
+purchase-workflow https://github.com/Eficent/purchase-workflow.git 11.0-mig-purchase_delivery_split_date
```

Comment:
dependency has ben merged :tada: Can you rebase and delete this?

Reply:
Done